about.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./about.css">
+    <script src="https://kit.fontawesome.com/d9a272ac0d.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;300;400;700&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400&family=Poppins:wght@100&family=Roboto:wght@100;700&display=swap" rel="stylesheet">
+    <title>Confrence</title>
+</head>
+<body id="body">
+    <section class="header">
+        <header class="head">
+            <nav class="nav-bar">
+                <a id="logo" href="#"><i class="fa-sharp fa-solid fa-bars menu scroll"></i></a>
+                <div class="mobile-menu">
+                  <div class="close-menu"><button class="menu-button"><img src="./images/close.png" alt="close-button" ></button></div>
+                  <ul class="mobnav">
+                      <li class="mobnav1" onclick="showhide()"><a class="moblink" href="./index.html">home</a></li><hr>
+                      <li class="mobnav1" onclick="showhide()"><a class="moblink" href="#">program</a></li><hr>
+                      <li class="mobnav1" id="mobnav" onclick="showhide()"><a class="moblink" href="#">Join</a></li><hr>
+                      <li class="mobnav1" id="mobnav" onclick="showhide()"><a class="moblink" href="#">Sponser</a></li><hr>
+                      <li class="mobnav1" id="mobnav" onclick="showhide()"><a class="moblink" href="#">News</a></li><hr>
+                    </ul>
+              </div>
+                <div class="desk-links">
+                    <ul class="deskul">
+                        <li class="desklist"><a href="#" class="desklink"><i class="fa-brands fa-facebook-f"></i></a></li>
+                        <li class="desklist"><a href="#" class="desklink"><i class="fa-brands fa-twitter"></i></a></li>
+                        <li class="desklist"><a href="#" class="desklink">English</a></li>
+                        <li class="desklist"><a href="#" class="desklink">My page</a></li>
+                        <li class="desklist"><a href="#" class="desklink">Logout</a></li>
+                    </ul>
+                </div>
+                <div class="desk-nav">
+                    <div class="logolink">
+                      <a class="logolink1" href="#"><span class="desklogo">VR Odyssey</span></a>
```

Comment:
- [OPTIONAL] Kindly consider giving this logo hyperlink to the homepage by placing the value `./index.html` in the href attribute.
I will highly recommend you to apply this because this requirement got missed by the previous reviewer.